### DISCUSSION TITLE
where search themes  and install a theme ?

### DISCUSSION THREAD

**Discussion by sergiohdljr** [fish-shell/fish-shell#10225] [Q&A]
I am new to fish, and would like to change the visual of the shell like in others termianals like oh-my-zsh

**Comment by faho** [comment]
"Theme" here can refer to two things:

1. A color scheme
2. A prompt - this is the text it displays whenever it's back in charge after a command

Fish ships with a few different of both and calls prompts "prompts" and colorschemes "themes".

To view those, run `fish_config` to see them in a browser, or `fish_config theme show` and `fish_config prompt show` to view them in your terminal.

Or you can just probably just search github for "fish theme" or "fish prompt". I know the [tide prompt](https://github.com/IlanCosman/tide) is pretty popular.

Or you can create your own, see [Writing Your Own Prompt](https://fishshell.com/docs/current/prompt.html).

**Reply by sergiohdljr** [reply]
oh yes i could have been more specific, i'm talk about color scheme. 

i will look to the options you sugest.

thank you.
